dll/win32/wininet/urlcache.c

```diff
@@ -1,8 +1,3 @@
-#ifdef __REACTOS__
-#define NONAMELESSUNION
-#define NONAMELESSSTRUCT
-#include "precomp.h"
-#else
```

Comment:
All the other wininet files use the precomp.h in this instance. Isn't it possible to keep this here as well?
Maybe the NONAMELESSUNION/STRUCT thingies should be moved into the precomp.h ?

Reply:
`NONAMELESSUNION` and `NONAMELESSSTRUCT` conflict with `internet.c`. Reverted these lines.